### ISSUE TITLE
trained VNTR models for Nanopore data

### ISSUE THREAD

**Issue by nbargues** [mehrdadbakhtiari/adVNTR#33]
Hi, when I download vntr_data_recommended_loci.zip , there are two files ( illumina and pacbio ). If I have Nanopore data, can I choose the pacbio file or do you have a specific nanopore file that I can find ?

**Comment by mehrdadbakhtiari** [comment]
Hi, we haven't extensively tested and measured the accuracy on Nanopore data. However, you can use the file related to pacbio and it should work reasonably well. Although they have relatively different error profile, given the long read in both there can be long run of well aligned basepairs so you can try to run the tool with pacbio data and `--pacbio` argument.

**Comment by nbargues** [comment]
Thanks for the response. 
So you suggest to use --pacbio argument, even if I have Nanopore data and that you have a --nanopore argument ?

**Comment by mehrdadbakhtiari** [comment]
Yes, `--nanopore` is not fully tested and current version won't let the code run. But I assume by pacbio parameters you can still get a relatively good results and it worth trying.
